core/src/main/java/org/infinispan/manager/DefaultCacheManager.java

```diff
@@ -773,11 +778,46 @@ private void internalStart(boolean block) {
       try {
          globalComponentRegistry.getComponent(CacheManagerJmxRegistration.class).start();
          globalComponentRegistry.start();
+         // Some caches are started during postStart and we have to extract it out
+         globalComponentRegistry.postStart();
```

Comment:
Chicken and egg problem. If a component verifies if the manager is running, would it fail? However, we can't set it to RUNNING before finishing the post-start.